src/file.js

```diff
@@ -5,10 +5,12 @@ export default (session, fileName) => {
   const file = {
     session,
     fileName,
+
     variables() {
       return this.session.client()
-        .then((client) => client.session.call('file.netcdf.variables', [fileName]));
+        .then((client) => client.session.call('cdat.file.variables', [fileName]));
```

Comment:
Funny how the linter behaves differently for you. @jbeezley Why is that? But I like your version better.